pyirf/interpolation/component_estimators.py

```diff
@@ -425,16 +425,17 @@ def __call__(self, target_point):
         -------
         aeff_interp: np.ndarray of (astropy.units.m)**2, shape=(n_points, ...)
             Interpolated Effective area array with same shape as input
-            effective areas. For AEFF2D of shape (n_energy_bins, n_fov_offset_bins)
-
+            effective areas. For AEFF2D of shape (n_energy_bins, n_fov_offset_bins). 
+            Values lower or equal to __init__'s min_effective_area are set 
+            to zero.
         """
 
         aeff_interp = super().__call__(target_point)
 
         # exp it and set to zero too low values
         aeff_interp = np.exp(aeff_interp)
-        # 1.1 to correct for numerical uncertainty and interpolation
-        aeff_interp[aeff_interp < self.min_effective_area * 1.1] = 0
+        # remove entries manipulated by min_effective_area 
+        aeff_interp[aeff_interp <= self.min_effective_area] = 0
```

Comment:
I'd expect a `<` here, I know it's floating point so it doesn't really matter, but this now means that things that are exactly `min_effective_area` are changed, which is surprising I think.

Reply:
> which violates the in between templates test

That test is wrong then, it should take the `min_effective_area` into account.